### ISSUE TITLE
Investigate 500 return code on DELETE of task

### ISSUE THREAD

**Issue by mfeit-internet2** [perfsonar/pscheduler#294]
Need to look at why we return 500 on DELETE of a task that's not there.

Casually related to https://github.com/perfsonar/mesh-config/issues/55.

From Slack:

```
vvidic [6:19 AM] 
Feb  8 12:17:34 pstest pscheduler-api ERROR    Exception: InternalError: Task not found while disabling task d397050d-da75-43a9-bf5c-afb9854f3120#012CONTEXT:  PL/pgSQL function api_task_disable(uuid,text) line 12 at RAISE#012#012Traceback (most recent call last):#012  File "/usr/share/pscheduler/api-server/pschedulerapiserver/dbcursor.py", line 97, in dbcursor_query#012    cursor.execute(query, args)#012InternalError: Task not found while disabling task d397050d-da75-43a9-bf5c-afb9854f3120#012CONTEXT:  PL/pgSQL function api_task_disable(uuid,text) line 12 at RAISE

[6:19]  
any way to clean up this task?

vvidic [6:25 AM] 
198.128.151.30 - - [08/Feb/2017:12:23:43 +0100] "DELETE /pscheduler/tasks/d397050d-da75-43a9-bf5c-afb9854f3120 HTTP/1.1" 500 1909 "-" "python-requests/2.4.3 CPython/2.7.9 Linux/3.16.0-4-amd64"

[6:25]  
maybe handle this rather than returning 500?

mfeit [7:41 AM] 
Not sure why that's happening.  The request is coming from the outside (presumably meshconfig), so the question would be why there's a difference between what MC and pSc think exists.  404 instead of 500 would be a better return for that.  I'll write a ticket on it.
```

**Comment by igarny** [comment]
This is related or similar to report #290

**Comment by mfeit-internet2** [comment]
This was fixed in 826548c2a37fd726bd6f465f84cee44d1e06c075.
